types/theme-ui/index.d.ts

```diff
@@ -1,29 +1,26 @@
-// Type definitions for theme-ui 0.2
+// Type definitions for theme-ui 0.3
 // Project: https://github.com/system-ui/theme-ui#readme
 // Definitions by: Erik Stockmeier <https://github.com/erikdstock>
 //                 Ifiok Jr. <https://github.com/ifiokjr>
 //                 Brian Andrews <https://github.com/sbardian>
 //                 Rodrigo Pombo <https://github.com/pomber>
 //                 Justin Hall <https://github.com/wKovacs64>
 //                 Prateek Kathal <https://github.com/prateekkathal>
+//                 Piotr Monwid-Olechnowicz <https://github.com/hasparus>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 3.1
 
-import { ResponsiveStyleValue, SystemStyleObject } from '@styled-system/css';
+import { SystemStyleObject } from '@styled-system/css';
 import * as CSS from 'csstype';
 import * as React from 'react';
-import { lineHeight, Theme as StyledSystemTheme } from 'styled-system';
+import { Theme as StyledSystemTheme } from 'styled-system';
 import { Interpolation, SerializedStyles } from '@emotion/serialize';
+export * from '@theme-ui/components';
```

Comment:
I am not sure why this belongs here. The goal is to only make types exportable, not components by itself.

Reply:
Take a look at https://github.com/system-ui/theme-ui/blob/b0257f0aa3dda6e65482552780fda715783205c3/packages/theme-ui/src/index.js#L18

This is the **title of my pull request**.
ThemeUI 0.3 reexports @theme-ui/components. We want to provide typings for these reexported components, so we can have typechecking and autoimports while using theme-ui.

Reply:
@hasparus Just to be clear, re-exporting can also be understood in terms of making improvements. Also, theme-ui exporting it's own components package directly is completely different than a types package exporting a theme-ui package.

Again, like I said... this package is only meant to define & export types, not the whole library or components. If this is the case, then this PR doesn't make sense at all. The package is meant to export only types not the whole components itself. 

Also, just to be clear, theme-ui doesn't support types locally, so there is no way type checking can happen. We have to manually define them. Removing them altogether defeats the purpose of this package itself.

Reply:
@hasparus It seems I completely overlooked the fact that a types/theme-ui components was released by you. 

Hmm... I think in that case, my comments are irrelevant! 

Thanks for making the PR 👍

Reply:
Sorry I didn’t express myself clearly. I assumed "types" is implicit in the context of DefinitelyTyped. A bit lazy of me.